src/components/Reports/FourteenMonthReports/Layout/Table/TableHead/TableHead.tsx

```diff
@@ -9,18 +9,19 @@ import {
   Typography,
 } from '@material-ui/core';
 // eslint-disable-next-line import/extensions
-import { FourteenMonthReportQuery } from '../GetFourteenMonthReport.generated';
+import { FourteenMonthReportQuery } from '../../../GetFourteenMonthReport.generated';
 import { TableHeadCell } from './TableHeadCell/TableHeadCell';
 
 export type Unarray<T> = T extends Array<infer U> ? U : T;
```

Comment:
Does `ElementOf` work here? https://github.com/krzkaczor/ts-essentials#elementof-type

Reply:
I am a bit afraid `ElementOf` would not work since the type needs to be `Maybe`.

Reply:
Wrap it in `Maybe`? 🤷‍♂️